src/RecordComponent.cpp

```diff
@@ -103,6 +104,21 @@ RecordComponent::flush(std::string const& name)
                 dCreate.compression = m_dataset->compression;
                 dCreate.transform = m_dataset->transform;
                 IOHandler->enqueue(IOTask(this, dCreate));
+                if( m_chunks->empty() )
+                {
+                    /* Ensure at least one WRITE_DATASET per dataset occurs
+                     * ADIOS1 backend only creates a dataset after at least once cell has been written */
```

Comment:
do we or ADIOS define the term cell?

Reply:
Interesting. ADIOS *does* use the same lingo for what I called cell here. (See e.g. [```adios_define_mesh_unstructured```](https://users.nccs.gov/~pnorbert/ADIOS-UsersManual-1.13.1.pdf) on page 52).
I used it independently here, though.